### PR TITLE
refactor: echo getting started page

### PR DESCRIPTION
**Pull request by davidsoderberg**
Screenshot 2024-05-03 at 10 00 55

### CODE REVIEW

File: apps/web/src/pages/get-started/components/get-started-tabs/GetStartedTabs.tsx [hunk 1]
```diff
@@ -33,6 +35,13 @@ export const GetStartedTabs: React.FC<IGetStartedTabsProps> = ({ tabConfigs = TA
         mb={15}
       >
         <Tabs.List>
+          <Tabs.Tab
+            key={`tab-${OnboardingUseCasesTabsEnum.ECHO}`}
+            value={OnboardingUseCasesTabsEnum.ECHO}
+            icon={<IconConnectedTv style={ICON_STYLE} />}
+          >
+            Echo Studio
```
Comment (scopsy) [inline apps/web/src/pages/get-started/components/get-started-tabs/GetStartedTabs.tsx:43]: I think we should remove Studio from here. Studio is a mere detail here. The main point is Code-First ala Echo only. Pretty sure we have adjusted this on Figma. Is that the latest copy?

Comment (davidsoderberg) [reply apps/web/src/pages/get-started/components/get-started-tabs/GetStartedTabs.tsx:43]: We did but Radek liked to have the studio there and now the latest is Workflows will push it when done with other changes.

**Review by antonjoel82 — COMMENTED** [review]
Thanks for the refactor here, David! There's a number of places that we're doing extra work in `EchoTab.tsx` by not using our tokens or some of the Panda utilities.

I scheduled a pair programming session after your holiday for us to refactor the component together since I think it will be easier and a good thing to record if you're okay with it! It would be helpful for me to hear any confusions or frustrations with dev with Panda so that we can document them and improve them for everyone :)

**Review by antonjoel82 — APPROVED** [review]
Nice work! There's a couple more time-savers / short-hands we can discuss later, but they're non-blocking

**Review by scopsy — APPROVED** [review]

**Comment by davidsoderberg** [comment]
A lot of changes have happened with the page since this pr was created please see new image on top and review the new changes :)

File: apps/web/src/pages/get-started/components/get-started-tabs/EchoTab.tsx [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+  });
+
+  return (
+    <Flex className={className} direction="row" alignItems="center" gap="300">
+      <div>
+        <Title variant="section" className={css({ marginTop: '100' })}>
+          Create notification workflows as code
+        </Title>
+        <Text variant="secondary" className={cx(css({ marginTop: '50' }), mainText)}>
+          With Novu, you to write notification workflows in your codebase locally right in your IDE and preview and edit
```
Comment (scopsy) [inline apps/web/src/pages/get-started/components/get-started-tabs/EchoTab.tsx:69]:
```suggestion
          With Novu, you write notification workflows in your codebase locally right in your IDE and preview and edit
```

**Review by scopsy — APPROVED** [review]

**Pull request merged**